app/components/discovery-panel/navigation-settings.tsx

```
@@ -1,26 +1,38 @@
-import { Accessibility, Building2, Map } from "lucide-react";
+import { Building2, Map } from "lucide-react";
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)